Automated workflow

helenst opened the pull request:
Automated workflow option and default location, from JiscRDSS

Bringing in #667 and #669 together as the dependency between these two fixes is quite tight.

sevein committed:
MCPServer: add automatedProcessingConfig.xml

sevein committed:
Dashboard: add default location option

Depends on: https://github.com/JiscRDSS/archivematica-storage-service/pull/8.
Related issue: #18.

sevein committed:
MCPServer: default AS location in automated wfl

sevein committed:
MCPClient: include default location choice

sevein committed:
Fix migration: 0038 -> 0039

sevein committed:
Fix typo

sevein committed:
fix permission bit

sevein committed:
Ignore static/

sevein reviewed (changes requested):
The automated workflow doesn't seem to include choices for new steps in qa/1.x like `Perform policy checks on originals`, `Perform policy checks on preservation derivatives`, etc...

helenst commented:
@sevein Thanks! I've rebased against qa/1.x and added those policy check steps to both automated and default workflows (they were missed when that got moved). Github won't let me reopen this and seems confused about the commits so I'll open a new PR.
